README.md

```diff
@@ -113,6 +113,21 @@ then in your component or controller 's actions:
   }
 ```
 
+### Skipping real library download
+
+Sometimes you want to have the behavior of a reCAPTCHA but without the burden of injecting the downloaded library into your web page. This is particularly useful during tests.
```

Comment:
@sinankeskin feel free to suggest improvements :)

Reply:
btw: I'm not sure if needs a CHANGELOG entry about that... 🤔

Reply:
I don't think so.